.github/workflows/ci-linux_mac.yml

```diff
@@ -90,6 +90,7 @@ jobs:
           # Bypass Catch2 Framework stdout interception with awk on test output
           ./tiledb/test/tiledb_unit -d yes | awk '/1: ::set-output/{sub(/.*1: /, ""); print; next} 1'
           ./tiledb/test/regression/tiledb_regression -d yes | awk '/1: ::set-output/{sub(/.*1: /, ""); print; next} 1'
+          ./tiledb/test/ci/test_assert -d yes | awk '/1: ::set-output/{sub(/.*1: /, ""); print; next} 1'
```

Comment:
Do these (::set-output) accumulate properly, or could success/failure both be seen, maybe success occurring after failure occurred earlier?